xla/service/buffer_assignment.cc

```diff
@@ -2108,7 +2124,12 @@ BufferAssigner::CreateAssignment(
   for (const auto& [color, computations] : private_stacks) {
     private_stack_colors.insert(color);
   }
-  assignment->CombineTempAllocations(private_stack_colors);
+  const bool temp_buffer_use_async_allocator =
```

Comment:
This should not be here as well, this should be passed from GPU compiler via arguments

Reply:
Fixed